store/src/server/api.rs

```diff
@@ -1,39 +1,65 @@
-use std::net::ToSocketAddrs;
-use tracing::info;
-use anyhow::Result;
-
 use crate::config::StoreConfig;
-use tonic::{transport::Server, Response, Status, Streaming};
-
+use crate::db::Db;
+use anyhow::Result;
+use miden_crypto::merkle::TieredSmt;
+use miden_crypto::{Felt, FieldElement};
 use miden_node_proto::store;
+use tokio::time::Instant;
+use std::net::ToSocketAddrs;
+use tonic::{transport::Server, Response, Status, Streaming};
+use tracing::{instrument, info};
```

Comment:
Similar nit re grouping imports from the same create together.

Reply:
done